content/docs/gto/user-guide/storages.md

```diff
@@ -0,0 +1,3 @@
+## Working with large artifacts
+
+...Git-lfs, DVC and what else is there...
```

Comment:
Could you please check out this page? https://dvc.org/doc/command-reference/install
Does it solve the problem?

Didn't use the tools that article mentions. How is that connected? Do you want to store secrets with DVC? Don't think I got what you mean :)

Reply:
Nice! Thanks, didn't know it :) this solves the problem of running the dvc commands on top of the git commands. I would recommend to hint to do so in this page.

> Do you want to store secrets with DVC? 

Not sure if with DVC, but it would be awesome if they could be "integrated" with the git repo.
In the [article I wrote](https://medium.com/@francesco.calcavecchia/we-refused-to-use-a-hammer-on-a-screw-story-of-a-gto-based-model-registry-c540ac5d129f) I explained it in the section "put: store a new artifact in the registry".
Should one ask each developer to set e.g. `AZURE_STORAGE_CONNECTION_STRING`? Is this the best practice?
It would be awesome if after a `git clone` of an artifact registry repo I could automatically get such credentials (if I am an authorized developer). And do so without storing unencrypted secrets.

I am not saying you must provide a solution, just mentioning that the secret sharing necessary to use a dvc artifact registry was one of the difficulties I encountered, and perhaps giving some hints would be useful.